src/System.Net.Http/src/netcore50/project.json

```diff
@@ -16,6 +16,7 @@
         "System.Resources.ResourceManager": "4.0.0",
         "System.Runtime.Extensions": "4.0.10",
         "System.Runtime.WindowsRuntime": "4.0.10",
+        "System.Security.Cryptography.X509Certificates": "4.1.0-rc3-24008-00",
```

Comment:
@bartonjs So, what is the next step? Are we blocked from rev'ing the System.Net.Http contract? Or do we just do it and work out the problems as we go along?